README.md

```diff
@@ -19,7 +19,6 @@ for updates.
 ## Requirements
 
 - [Sass](https://github.com/sass/sass) 3.3+
-- :warning: If you need **Sass 3.2 support**, you should [use Neat 1.5.1](#installing-older-versions-of-neat)
```

Comment:
I dropped support for Sass v3.2 in the v2.x lifecycle of the library. Seemed to make sense.